improver/utilities/cube_manipulation.py

```diff
@@ -211,7 +211,11 @@ def _check_time_bounds_ranges(cube):
                 raise ValueError(msg)
 
     def process(
-        self, cubes_in, check_time_bounds_ranges=False, slice_over_realization=False
+        self,
+        cubes_in,
+        check_time_bounds_ranges=False,
+        slice_over_realization=False,
+        copy=True,
```

Comment:
Can we have a unit test for this addition please. It is new behaviour for this plugin.